### DISCUSSION TITLE
Next.js app error when deploying to Vercel

### DISCUSSION THREAD

**Discussion by devSahinur** [vercel/community#487] [Help] [answered]
Hi I tried npm run build or yarn run build
everything works at my local machine

But when I tried to deploy it to Vercel, this error shows up
```
Module not found: Can't resolve '../components/layout' in '/vercel/path0/pages'
```

**Comment by samcx** [comment, marked as answer]
You are liking coming across this error → https://vercel.com/support/articles/how-do-i-resolve-a-module-not-found-error. Let us know if that helps!
